test/remote/gateways/remote_litle_test.rb

```diff
@@ -421,11 +427,11 @@ def test_authorize_and_capture_with_stored_credential_recurring
       }
     )
     assert auth = @gateway.authorize(4999, credit_card, initial_options)
-    assert_success auth
-    assert_equal 'Approved', auth.message
+    assert_failure auth
+    assert_equal 'Transaction Received', auth.message
     assert network_transaction_id = auth.params['networkTransactionId']
 
-    assert capture = @gateway.capture(4999, auth.authorization)
+    assert capture = @gateway.capture(4999, auth.authorization, initial_options)
```

Comment:
💬  Comment:

I'm seeing a bunch of caputre, void and refund calls that now have intial_options | used_options, any particular reason to add them ?

the tests worked before because the base for  capture/void/refund call is the authorization string that should have the needed references reverse a transaction.

Reply:
The options now are required because the new version requires the field orderId in the XML request so the options are being used to attach it in every request.